Updated the docker deployment for two services and update the README.md file with setup guide. (#41)
yashodgayashan/environment_platform: README.md

> # environment_platform

Reviewer: ```suggestion
# Environment Platform
```

> ## Core

Reviewer: ```suggestion
## Module Core(contains the back-end functionalities).
```

The following instruction must be followed to setup the backend in your local machine

> ### Requirments

Reviewer: ```suggestion
### Prerequisites
```

- Ballerina 1.2
- Docker

> ### Build

Reviewer: ```suggestion
### How to Build?
```

To build the Core please execute the following command

```
./setup.sh
```

> ### Configuration file

Reviewer: ```suggestion
### Create a configuration file
```

> Please Make a file called `ballerina.conf` in core/resources as given in the `example.conf` and update the values.

Reviewer: ```suggestion
Make a file named `ballerina.conf` in `core/resources`. You can refer the file `example.conf` located at `core/resources/` for more details. Then, update the created file with your preference of values.
```

> ### Run the project using docker

Reviewer: ```suggestion
### Run the module using docker
```

All the volumes and internal networking for mongodb container is set using the docker-composer. Please follow the following commands to build the image and run.

##### Build the image

```
docker-compose build
```

##### Run the image

```
docker-compose up
```

##### Terminate the docker container

> - First press `ctl+c`.

Reviewer: ```suggestion
- Press `Ctrl+C` to terminate the program.
```

> - Then run `docker-compose down`

Reviewer: ```suggestion
- Run `docker-compose down` to stop your containers.
```

#### Develop and run using docker

> When you develop a feature and run using containers. You have to build the jar files using `./setup.sh` and then you have to build and up the dokcer composer.

Reviewer: ```suggestion
When you develop a feature and run using containers. You have to build the jar files using `./setup.sh` and then you have to build and up the docker composer.
```

# Run the project using docker

> **Important** - Make sure that configuration files are setup for local machine.

Reviewer: ```suggestion
**Important** - Make sure that the configuration files are setup for your local machine.
```

To run the project,

> - First `cd core`.

Reviewer: ```suggestion
- Execute the command `cd core` to navigate to the `core` module.
```

> - Then run `./scripts/run-local.sh`.

Reviewer: ```suggestion
- Then, run the script using the command `./scripts/run-local.sh`.
```

> # Run the project using docker

Reviewer: Isn't this run the project locally? And if it is:
```suggestion
### Run the project locally
```

> To run the project,

Reviewer: ```suggestion
To run the module,
```
